README.md

```diff
@@ -35,36 +35,64 @@ github "DevCycleHQ/ios-client-sdk"
 
 ### Initializing the SDK
 
+#### Swift
 Using the builder pattern we can initialize the DevCycle SDK by providing the DVCUser and DevCycle mobile environment key:
 
 ```swift
 let user = try DVCUser.builder()
                     .userId("my-user1")
                     .build()
 
-guard let client = try DVCClient.builder()
+guard let dvcClient = try DVCClient.builder()
         .environmentKey("<DEVCYCLE_MOBILE_ENVIRONMENT_KEY>")
         .user(user)
         .build(onInitialized: nil)
 ```
 
 The user object needs either a `user_id`, or `isAnonymous` set to `true` for an anonymous user. 
 
+#### Objective-C
+For Objective-C we use a standard callback pattern to initialize the DevCycle SDK by providing the DVCUser and DevCycle mobile environment key:
+
+```objc
+DVCUser *user = [DVCUser initializeWithUserId:@"my-user1"];
+
+
+self.dvcClient = [DVCClient initialize:@"<DEVCYCLE_MOBILE_ENVIRONMENT_KEY>"
+                               user:user
+                            options:nil
+                      onInitialized:^(NSError * _Nullable error) {
+    if (error) {
+        NSLog(@"DevCycle failed to initialize: %@", error);
+    }
+}];
+```
+
 ## Using Variable Values
 
 To get values from your Features, the `variable()` method is used to fetch variable values using 
 the variable's identifier `key` coupled with a default value. The default value can be of type 
 string, boolean, number, or JSONObject:
 
+#### Swift
 ```swift
-let strVariable: DVCVariable<String> = client.variable(key: "str_key", defaultValue: "default")
-let boolVariable: DVCVariable<Bool> = client.variable(key: "bool_key", defaultValue: false)
-let numVariable: DVCVariable<Int> = client.variable(key: "num_key", defaultValue: 4)
-let jsonVariable: DVCVariable<[String:Any]> = client.variable(key: "json_key", defaultValue: [:])
+let boolVariable = dvcClient.variable(key: "bool_key", defaultValue: false)
+let strVariable = dvcClient.variable(key: "string_key", defaultValue: "default")
+let numVariable = dvcClient.variable(key: "num_key", defaultValue: 4)
+let jsonVariable = dvcClient.variable(key: "json_key", defaultValue: [:])
+```
+
+#### Objective-C
+```objc
+DVCVariable *boolVariable = [self.dvcClient boolVariableWithKey:@"bool_key" defaultValue:false];
```

Comment:
@jsalaber thoughts on using `ForKey:` instead of `WithKey`?

Reply:
either works for me

Reply:
seems like `With` is mainly used: https://stackoverflow.com/questions/8410602/objective-c-method-naming-convention